libpod/container_inspect.go

```diff
@@ -0,0 +1,161 @@
+		Id:      config.ID,
+		Created: config.CreatedTime,
+		Path:    path,
+		Args:    args,
+		State: &CtrState{
+			OciVersion: spec.Version,
+			Status:     runtimeInfo.State.String(),
+			Running:    runtimeInfo.State == ContainerStateRunning,
+			Paused:     runtimeInfo.State == ContainerStatePaused,
+			OOMKilled:  runtimeInfo.OOMKilled,
+			Dead:       runtimeInfo.State.String() == "bad state",
+			Pid:        runtimeInfo.PID,
+			ExitCode:   runtimeInfo.ExitCode,
+			Error:      "", // can't get yet
+			StartedAt:  runtimeInfo.StartedTime,
+			FinishedAt: runtimeInfo.FinishedTime,
+		},
+		ImageID:         config.RootfsImageID,
+		ImageName:       config.RootfsImageName,
+		ResolvConfPath:  runtimeInfo.ConfigPath,                               // not sure
+		HostnamePath:    spec.Annotations["io.kubernetes.cri-o.HostnamePath"], // not sure
+		HostsPath:       "",                                                   // can't get yet
+		StaticDir:       config.StaticDir,
+		LogPath:         ctr.LogPath(),
+		Name:            config.Name,
+		Driver:          driverData.Name,
+		MountLabel:      config.MountLabel,
+		ProcessLabel:    spec.Process.SelinuxLabel,
+		AppArmorProfile: spec.Process.ApparmorProfile,
+		ExecIDs:         "", //can't get yet
+		HostConfig: &HostConfig{
+			Binds:                config.Volumes,
+			ConsoleSize:          spec.Process.ConsoleSize,
+			OomScoreAdj:          spec.Process.OOMScoreAdj,
+			Cgroup:               spec.Linux.CgroupsPath,
+			CpuShares:            shares,
+			BlkioWeight:          blkioWeight,
+			BlkioWeightDevice:    blkioWeightDevice,
+			BlkioDeviceReadBps:   blkioReadBps,
+			BlkioDeviceWriteBps:  blkioWriteBps,
+			BlkioDeviceReadIOps:  blkioReadIOPS,
+			BlkioDeviceWriteIOps: blkioeWriteIOPS,
+			CpuPeriod:            period,
+			CpuQuota:             quota,
+			CpuRealtimePeriod:    realtimePeriod,
+			CpuRealtimeRuntime:   realtimeRuntime,
+			CpusetCpus:           cpus,
+			CpusetMems:           mems,
+			Devices:              spec.Linux.Devices,
+			KernelMemory:         memKernel,
+			MemoryReservation:    memReservation,
+			MemorySwap:           memSwap,
+			MemorySwappiness:     memSwappiness,
+			OomKillDisable:       memDisableOOMKiller,
+			PidsLimit:            pidsLimit,
+			Priveleged:           spec.Process.NoNewPrivileges,
+			ReadonlyRootfs:       spec.Root.Readonly,
+			Runtime:              ctr.runtime.ociRuntime.name,
+		},
+		GraphDriver: driverData,
+		Mounts:      spec.Mounts,
+		Config: &CtrConfig{
+			Hostname:    spec.Hostname,
+			User:        spec.Process.User,
+			Env:         spec.Process.Env,
+			Image:       config.RootfsImageName,
+			WorkingDir:  spec.Process.Cwd,
+			Labels:      config.Labels,
+			Annotations: spec.Annotations,
+			Tty:         spec.Process.Terminal,
+			OpenStdin:   config.Stdin,
+			StopSignal:  config.StopSignal,
+			Cmd:         config.Spec.Process.Args,
+		},
+		NetworkSettings: &NetworkSettings{
+			SandboxID: config.Pod,
```

Comment:
Leave this blank for now, pods aren't really working yet.

Reply:
fixed